cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminDesignTime.feature

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Sysadmin
+Feature: Sysadmin - Validate system admin page design time scenarios
+
+  @Sysadmin @SysAdminRequired
+  Scenario:Validate user is able to create new system preferences and able to delete the added system preferences successfully
+    Given Open Datafusion Project to configure pipeline
+    When Open "System Admin" menu
+    Then Click on the Configuration link on the System admin page
+    Then Select "systemPreferences" option from Configuration page
+    Then Click on edit system preferences
```

Comment:
you can use `Background` section  to define common steps that are to be run for all the scenarios in the feature.
Refer https://cucumber.io/docs/gherkin/reference/#background for more details and examples

Reply:
Added the Background section for the common steps in features.